ui/packages/catalog-ui-search/src/main/webapp/js/CQLUtils.js

```diff
@@ -223,7 +224,9 @@ function generateFilter(type, property, value, metacardDefinitions) {
   switch (metacardDefinitions.metacardTypes[property].type) {
     case 'LOCATION':
     case 'GEOMETRY':
-      return generateAnyGeoFilter(property, value)
+      const geo = generateAnyGeoFilter(property, value)
+      geo.geojson = serialize(value)
```

Comment:
@andrewkfiedler not sure what key to use to store the geojson value. Do you like `geojson`?